.circleci/config.yml

```diff
@@ -1010,11 +1063,54 @@ jobs:
             ./Tests/scripts/slack_notifier.sh 'sdk_faild_steps' ./env_results.json
           when: always
 
+  Demisto SDK Nightly - Run against CortexXSOAR:
+    <<: *container_config
+    resource_class: medium
+    <<: *environment
+    steps:
+      - checkout
+      - setup_remote_docker
+      - *restore_cache
+      - *prepare_environment
+      - *download_configuration
+      - *create_instance
+      - *setup_instance
+      - run:
+          name: Wait until server ready
+          shell: /bin/bash
+          when: always
+          command: |
+            python3 ./Tests/scripts/wait_until_server_ready.py "Demisto Marketplace"
+      - run:
+          name: Set Environment Variables
+          when: on_success
+          command: |
+            echo 'export CIRCLE_ARTIFACTS="/home/circleci/project/artifacts"' >> $BASH_ENV
```

Comment:
you already have it in `prepare_environment`

Reply:
didn't work. We worked on it together.

Reply:
Just add it [here](https://app.circleci.com/settings/project/github/demisto/content/environment-variables) as variable, it's a constant, no need to add it in run time.